src/guessWords/containers/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useCallback } from "react";
+import Congrats from "../components/Congrats";
+import GuessedWords from "../components/GuessedWords";
+import Input from "../containers/Input";
+
+function App() {
+  const [success, setSuccess] = useState(false);
+  const [secretWord, setSecretWord] = useState("");
+  const [guessedWords, setGuessedWords] = useState([]);
+  const [matchcount, setMatchCount] = useState([]);
+  const guesswordCallback = useCallback(
+    (guessedWord) => {
+      if (guessedWord == secretWord) {
+        setSuccess(true);
+      }
+      setGuessedWords([
+        ...guessedWords,
+        {
+          guessedWord: guessedWord,
+          letterMatchCount: getLetterMatchCount(guessedWord, secretWord),
+        },
+      ]);
+    },
+    [guessedWords, secretWord]
+  );
+  useEffect(() => {
+    setSecretWord("Letter");
+  }, []);
+
+  const getLetterMatchCount = (guessedWord, secretWord) => {
+    let t = 0;
+    const min = Math.min;
+    for (let i = 0; i < 26; i++) {
+      t +=
+        min(
+          guessedWord.split(String.fromCharCode(i + 65)).length,
+          secretWord.split(String.fromCharCode(i + 65)).length
+        ) - 1;
+    }
+    for (let i = 0; i < 26; i++) {
+      t +=
+        min(
+          guessedWord.split(String.fromCharCode(i + 97)).length,
+          secretWord.split(String.fromCharCode(i + 97)).length
+        ) - 1;
+    }
+    return t;
+  };
+
+  const guessWord = guesswordCallback;
```

Comment:
if `guessWordCallback` is only used here, avoid creating a redundant variable and just create `guessWord` function.